homeassistant/components/hive.py

```diff
@@ -0,0 +1,707 @@
+"""Hive Integration - Platform."""

```

Comment:
Please use the same style for the file header as we do for other component.

Reply:
All files updated with the standard header style